tests/apollo/util/skvbc_history_tracker.py

```diff
@@ -46,6 +46,12 @@ async def wrapper(*args, **kwargs):
                 skvbc = kvbc.SimpleKVBCProtocol(bft_network)
                 init_state = skvbc.initial_state()
                 tracker = SkvbcTracker(init_state, skvbc, bft_network, pre_exec_enabled, no_conflicts)
+
+                if uses_real_blockchain:
+                    client = await bft_network.new_client()
+                    initial_block_id = await tracker.get_last_block_id(client)
+
+                tracker.set_initial_block_id(initial_block_id)
```

Comment:
is `initial_block_id` set before this is called if `uses_real_blockchain = False` ?

Reply:
Yes, it's set to 0 in the constructor. 

Ideally this check should have been in the constructor itself, but it's a chicken-egg problem. get_last_block_id() is part of the tracker and I think copy-pasting its implementation is worse than a setter.